src/dbnode/storage/shard_test.go

```diff
@@ -1908,21 +1877,8 @@ func TestShardAggregateTiles(t *testing.T) {
 		{thirdSourceBlockStart, volume2},
 	}
 
-	write1 := newWrittenDataMatcher(t, []ts.Datapoint{
-		dp(start.Add(time.Minute), 5),
-	}, xtime.Nanosecond, nil)
-
-	write2 := newWrittenDataMatcher(t, []ts.Datapoint{
-		dp(start, 0.5),
-		dp(start.Add(2*time.Minute), 2),
-		dp(secondSourceBlockStart.Add(2*time.Minute), 5+3),
-		dp(secondSourceBlockStart.Add(3*time.Minute), 0),
-		dp(secondSourceBlockStart.Add(4*time.Minute), 9),
-	}, xtime.Nanosecond, counterPayload)
-
-	write3 := newWrittenDataMatcher(t, []ts.Datapoint{
-		dp(secondSourceBlockStart.Add(time.Minute), 3),
-	}, xtime.Nanosecond, gaugePayload)
+	targetShard := testDatabaseShardWithIndexFn(t, testOpts, nil, true)
+	defer targetShard.Close() // nolint:errcheck
```

Comment:
nit; may be worth doing a 
```golang
defer func() {
 assert.NoError(t, targetShard.Close())
}()
```
here instead to ensure closing is always clean?

Reply:
Will do. I guess just `defer assert.NoError(t, targetShard.Close())` should work.